### PR TITLE
{lib}[GCCcore/8.2.0] PROJ v6.0.0

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aa842b7b092c1703c20e8adaf3a288cc for a full test report.

File: easybuild/easyconfigs/p/PROJ/PROJ-6.0.0-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+geographic longitude and latitude coordinates into cartesian coordinates"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://download.osgeo.org/proj/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4510a2c1c8f9056374708a867c51b1192e8d6f9a5198dd320bf6a168e44a3657']
+
+builddependencies = [('binutils', '2.31.1')]
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PROJ/PROJ-6.0.0-GCCcore-8.2.0.eb]: @Micket appears to be missing dependency on `SQLite3`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/45c29bfdb88a7dc60b803a1a7c296583 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/64c5888f752fd0b5f0f75a7fea3e784a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2411.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ea305b0c35977ae17a19ea3e45d5335a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**